CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/compare/v0.1.0-beta.1...HEAD)
 
+### Added
+
+- Adds MongoDB instrumentation support from .NET Core 3.1+.
```

Comment:
If we use the suggestion below, can we get rid of the customization on the markdown lint?
```suggestion
- MongoDB instrumentation support from .NET Core 3.1+.
```

Reply:
It doesn't like the "Added" as a title? Does want a name instead? Anyway, let's move ahead and refine it later.

Reply:
#638